sql/synth_recipes.sql

```diff
@@ -4704,6 +4704,8 @@ INSERT INTO `synth_recipes` VALUES (75508,1,0,0,98,0,0,0,0,0,0,4100,4242,12420,0
 INSERT INTO `synth_recipes` VALUES (75509,1,0,0,103,0,0,0,0,0,0,4100,4242,12548,0,0,0,0,0,0,0,855,1225,654,655,1,6,2,2,'Adaman Cuirass (desynth)'); -- use synth lvl, wiki results
 INSERT INTO `synth_recipes` VALUES (75510,1,0,0,100,0,0,0,0,0,0,4100,4242,12932,0,0,0,0,0,0,0,855,1225,654,655,1,6,2,1,'Adaman Sabatons (desynth)'); -- use synth lvl, wiki results
 INSERT INTO `synth_recipes` VALUES (75511,0,0,0,0,0,0,0,0,0,101,4096,4238,615,4356,5568,5569,5688,0,0,0,6601,6601,6602,6602,6,8,6,8,'Omelette Sandwich'); -- BG Wiki and JP Wiki
+INSERT INTO synth_recipes VALUES(75512,1,0,0,0,0,23,0,0,0,0,4100,0,12729,0,0,0,0,0,0,0,818,819,807,807,2,4,1,2, "Linen Cuffs (desynth)");
+INSERT INTO synth_recipes VALUES(75513,1,0,0,0,0,25,0,0,0,0,4100,0,12857,0,0,0,0,0,0,0,818,818,819,819,2,3,6,7, "Linen Slops (desynth)");
```

Comment:
hey see how the rows right above yours cite their sources? also see thos commente dout row for exactly what you are adding? https://github.com/LandSandBoat/server/blob/612595acd58f5c6b919fd4891bc1c87aae6dad1d/sql/synth_recipes.sql#L2661

also please see this issue I made and even pinned to the top of the issues page https://github.com/LandSandBoat/server/issues/885